$NetBSD: patch-configure,v 1.2 2012/04/20 07:24:39 dholland Exp $

- dragonfly support

- ignore result of opal_speexdsp_float test as it is not robust.
AFAICT in pkgsrc it should always produce "no", but it doesn't always,
and this leads to build failure later.


--- configure.orig	2009-09-22 00:58:43.000000000 +0000
+++ configure
@@ -4650,6 +4650,7 @@ echo "$as_me: WARNING: \"CPU $target_cpu
 esac
 case "$target_os" in
   linux*)             OSTYPE=linux;        ;;
+  dragonfly* )        OSTYPE=DragonFly;    ;;
   freebsd* )          OSTYPE=FreeBSD;      ;;
   openbsd* )          OSTYPE=OpenBSD;      ;;
   netbsd* )           OSTYPE=NetBSD;       ;;
@@ -4900,7 +4901,7 @@ STDCCFLAGS="$STDCCFLAGS -D__MACOSX__"
           HAVE_RANLIB=yes
         ;;
 
-        freebsd*|openbsd*|netbsd*)
+        dragonfly*|freebsd*|openbsd*|netbsd*)
           SHAREDLIBEXT=so
           SHARED_CFLAGS="-fPIC"
 
@@ -7135,7 +7136,7 @@ echo "${ECHO_T}no" >&6; }
               { echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6; }
               case "$target_os" in
-                freebsd*|openbsd*|netbsd*|darwin*)
+                dragonfly*|freebsd*|openbsd*|netbsd*|darwin*)
                   { echo "$as_me:$LINENO: checking for dlopen in -lc" >&5
 echo $ECHO_N "checking for dlopen in -lc... $ECHO_C" >&6; }
 if test "${ac_cv_lib_c_dlopen+set}" = set; then
@@ -9391,7 +9392,7 @@ echo $ECHO_N "checking Speex has float..
 echo "${ECHO_T}$opal_speexdsp_float" >&6; }
 
 
-          if test $opal_speexdsp_float = yes; then
+          if false && test $opal_speexdsp_float = yes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define OPAL_SPEEX_FLOAT_NOISE 1
